src/scripts/ads/vpaid/VPAIDFlashTech.js

```diff
@@ -28,6 +28,8 @@ function VPAIDFlashTech(mediaFile, settings) {
 
 VPAIDFlashTech.VPAIDFLASHClient = VPAIDFLASHClient;
 
+VPAIDFLASHClient.runFlashTest(); // test async in the background if flash is supported or not
```

Comment:
Will this method not run until we require VPAIDFlashTech???
Because that will only happen once we are checking if the Inline is supported and it will be already too late.